src/Standards/AcquiaEdge/ruleset.xml

```diff
@@ -8,11 +8,4 @@
 
   <description>Acquia's Edge (backwards-incompatible) coding standards.</description>
```

Comment:
Do we need to break this into `AcquiaPHPEdge` and `AcquiaDrupalEdge`? If it's really inclusive of both, I foresee the same conceptual conflicts here that we're eliminating everywhere else by this refactoring. See my overall review comment for more detailed thinking.

Reply:
Yeah. Let's resolve the overall discussion of whether we want to delete the Edge standards first.

Reply:
Marking this resolved per my agreement above that we should just eliminate `AcquiaEdge` altogether.